SonarQube.Client/Api/V5_10/GetIssuesRequest.cs

```diff
@@ -88,7 +88,10 @@ private static IEnumerable<T> ReadFromProtobufStream<T>(Stream stream, MessagePa
         }
 
         private static SonarQubeIssue ToSonarQubeIssue(ServerIssue issue) =>
-            new SonarQubeIssue(FilePathNormalizer.NormalizeSonarQubePath(issue.Path), issue.Checksum, issue.Line, issue.Msg, issue.ModuleKey, issue.RuleKey,
-                issue.Status == "RESOLVED", flows: null);
+            new SonarQubeIssue(FilePathNormalizer.NormalizeSonarQubePath(issue.Path), issue.Checksum, issue.Msg,
+                issue.ModuleKey, issue.RuleKey, issue.Status == "RESOLVED",
+                severity: SonarQubeIssueSeverity.Unknown,
```

Comment:
Technically the `severity` field is available, but this is such an old SQ version that I didn't bother (as this field is needed for taint and that will be for 7.2+ anyway).